### PR TITLE
Add new extensions

### PR DESCRIPTION
**Pull request by omaralbeik**
- deprecate duplicated methods in **SwifterSwift.swift**
- new **Date** extensions
- new **URLRequest** extensions
- new **UIWebView** extensions
- new **UserDefaults** extensions
- updated **CHANGELOG**

### CODE REVIEW

**Comment by SD10** [comment]
Dang, I was hoping there was the date extensions I needed. I need to figure out if a date is in this week or in this year 😃 I'll add them after I complete it

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,40 @@ All notable changes to this project will be documented in this file.
 > ### Bugfixes
 > N/A
 
+# v3.2.0
+
+- **SwifterSwift**
```
Comment (SD10) [inline CHANGELOG.md]: Can we label this section **API Breaking**?

File: Sources/Extensions/Deprecated/SwifterSwiftDeprecated.swift [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+//
+//
+
+import Foundation
+
+// MARK: - Preperties
+public extension SwifterSwift {
+	
+	@available(*, deprecated: 3.2.0, message: "Use Apple's UserDefaults.standard instead")
+	/// SwifterSwift: Shared instance of standard UserDefaults (read-only).
```
Comment (SD10) [inline Sources/Extensions/Deprecated/SwifterSwiftDeprecated.swift]: Ok I agree with removing this

File: Sources/Extensions/Foundation/UserDefaultsExtensions.swift [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import Foundation
+
+
+// MARK: - Methods
+public extension UserDefaults {
+	
+	/// SwifterSwift: get object from UserDefaults by using subscript
+	///
+	/// - Parameter key: key in the current user's defaults database.
+	public subscript(key: String) -> Any? {
```
Comment (SD10) [inline Sources/Extensions/Foundation/UserDefaultsExtensions.swift]: 👍

File: CHANGELOG.md (hunk 1 above)
Comment (omaralbeik) [reply CHANGELOG.md]: Sure!

**Pull request merged**